community/syslog-summary/APKBUILD

```diff
@@ -0,0 +1,55 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=syslog-summary
+pkgver=1.14
+pkgrel=0
+pkgdesc="Summarize the contents of a syslog log file / useful with logcheck"
+url="https://packages.debian.org/sid/syslog-summary"
+arch="noarch"
+license="GPL3"
+depends="py-magic"
+makedepends=""
+install=""
+subpackages="$pkgname-doc"
+source="http://ftp.debian.org/debian/pool/main/s/$pkgname/${pkgname}_$pkgver.orig.tar.gz
+	00-fix_Makefile.patch
+	01-fix_Shebang.patch
+	02-new-python-magic.patch
+	03-socklog-compat.patch
+	04-rm-whitespace.patch
+	"
+builddir="$srcdir"/$pkgname-$pkgver
+
+prepare() {
+        local i
+	cd "$builddir"
+        for i in $source; do
+                case $i in
+                *.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+                esac
+        done
+}
```

Comment:
Please remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.